### DISCUSSION TITLE
Quick draw badge

### DISCUSSION THREAD

**Discussion by ghost** [community/community#44898] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Tried to get quickdraw multiple times. Each of my pull requests were accepted within 5 minutes but the badge ds not show

**Comment by mrowan137** [comment]
When I click on your profile, it seems to be showing up on my end (earned January 20th): ![image](https://user-images.githubusercontent.com/38045958/213762175-11042ea3-2136-42ae-9acf-025f364eb069.jpeg)
